client/src/App.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <v-app id="app">
+    <div id="main-column">
+      <Header />
+      <router-view />
+    </div>
+  </v-app>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+import Header from '@/components/header.vue'
+
+@Component({
+  components: { Header }
+})
+export default class App extends Vue {}
+</script>
+<style lang="stylus">
+  @import './assets/styles/variables.styl'
+  @import './assets/styles/base.styl'
```

Comment:
Instead of importing the main style files here for the whole app, we import them in main.ts, after the vuetify import, so that import order (and overrides) are correctly handled. IMO it feels ugly to do it that way but it works, and this doesn't.